utils/build/docker/nodejs/google_genai_app/app.js

```diff
@@ -0,0 +1,117 @@
+const tracer = require('dd-trace').init({});
+
+const express = require('express');
+
+const app = express();
+app.use(express.json({ limit: '50mb' }));
+app.use(express.urlencoded({ limit: '50mb', extended: true }));
+
+const { GoogleGenAI } = require('@google/genai');
+const genai = new GoogleGenAI({
+  httpOptions: {
+    baseUrl: `${process.env.DD_TRACE_AGENT_URL}/vcr/genai`
+  }
+});
+
+
+app.post('/generate_content', async (req, res) => {
+  const { model, contents, config = {} } = req.body;
+
+    const stream = config.stream;
+    delete config.stream;
+
+    const options = {
+      model,
+      contents: normalizeGoogleGenAiContents(contents),
+      config: normalizeGoogleGenAiConfig(config)
+    }
+
+    let response;
+
+    if (stream) {
+      response = [];
+      const stream = await genai.models.generateContentStream(options);
+      for await (const chunk of stream) {
+        response.push(chunk);
+      }
+    } else {
+      response = await genai.models.generateContent(options);
+    }
+
+    res.json({ response });
+});
+
+app.post('/embed_content', async (req, res) => {
+  const { model, contents } = req.body;
+  const response = await genai.models.embedContent({ model, contents });
+  res.json({ response });
+});
+
+function normalizeSnakeCaseConfigToCamelCase (generationConfig) {
+  const normalizedConfig = {};
+  for (const key of Object.keys(generationConfig)) {
+    // turn keys into camelCase
+    const camelKey = key.replace(/_([a-z])/g, (_, p1) => p1.toUpperCase());
+    normalizedConfig[camelKey] = generationConfig[key];
+    if (key !== camelKey) {
+      delete normalizedConfig[key];
+    }
+  }
+
+  return normalizedConfig;
+}
+
+function normalizeGoogleGenAiContents (contents) {
```

Comment:
these can probably be cleaned up somehow, just copied this over from the old shared tests